config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+resources :cinema_hall, only: [:index, :new, :create, :show, :update, :edit, :destroy]
```

Comment:
Your controller is named `cinema_halls` and route is for `cinema_hall`. This would produce an error